repos/fdbt-site/cypress_tests/cypress/integration/myFares/myFares.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    addFlatFareProductIfNotPresent,
+    clickElementById,
+    clickElementByText,
+    clickRandomElementsInTable,
+    getHomePage,
+} from '../../support/helpers';
+
+describe('Able to access services in my fares', () => {
+    it('Able to access service pages', () => {
+        getHomePage();
+        clickElementById('manage-fares-link');
+        clickRandomElementsInTable('govuk-table__body', 'service-link');
```

Comment:
Do we need an assert inside here or inside the function?

Reply:
And inside the other tests, if so

Reply:
So by clicking through the pages you're asserting that that journey works - if any elements aren't present/redirection doesn't occur correctly, it'll fail. That sound alright?